### DISCUSSION TITLE
no more add/remove device button

### DISCUSSION THREAD

**Discussion by lebphi** [zwave-js/zwave-js-ui#3106] [General]
hi,

my add/remove device button have disappeared .... I think i have perhaps made an error in the setting.json file. 
What is the option for retreive this menu ?

thank you

**Comment by Shakey001** [comment]
I lost add/remove device button also, had a SD card corrupt, rebuilt, system, updated to latest version, same issue. Tried older version no luck, reset Zooz ZST39 (800series) stick with Simplicity Studio still no luck.  Even tried an older 500 series stick that was laying around, no luck. I get a warning could not find scenes.json during docker compose up. Not sure how to recreate file if issue.

**Comment by lebphi** [comment]
i've a file scenes.json but it not resolve the problem.

I've compare old settings.json when i had the add/remove button and the new one where i haven't this button but i don't find the diffrence.

if i can't retreive the add/remove button i've read it is possible to include/remove device by mqtt command. Do you have examples for that ?

**Comment by kpine** [comment]
Are you looking in the right place? The manage nodes dialog moved a few versions ago.

![image](https://github.com/zwave-js/zwave-js-ui/assets/791794/ade3ee67-2e18-4c94-bc66-439322a36286)  
![image](https://github.com/zwave-js/zwave-js-ui/assets/791794/5e4d89a8-11e7-4274-8856-2f44254f46d4)

**Comment by Shakey001** [comment]
Thanks, refreshed the container, did a fresh compose up and works now.

**Comment by lebphi** [comment]
Effectively you have right sorry
